packages/react-google-analytics/src/tests/GaJS.test.tsx

```diff
@@ -206,6 +192,32 @@ describe('<GaJS />', () => {
     });
   });
 
+  describe('disableTracking', () => {
+    it('not setting disableTracking should load script without restrictions', () => {
+      const gajs = mount(<GaJS {...mockProps} />);
+      expect(gajs.find(ImportRemote).prop('source')).toBe(GA_JS_SCRIPT);
+      expect(
+        gajs.find('script').prop('dangerouslySetInnerHTML'),
+      ).toHaveProperty('__html', SETUP_SCRIPT);
+    });
+
+    it('true', () => {
```

Comment:
`it` blocks should be full sentences, If this failed in CI it would be hard to tell what it had been testing.